nemo/collections/tts/modules/radtts.py

```diff
@@ -652,10 +661,15 @@ def infer(
             dur = dur[:, 0]
             dur = dur.clamp(0, token_duration_max)
 
-        # get attributes f0, energy, vpred, etc)
```

Comment:
Why did you have to revert my changes? We should be using the math in regulate_len, not duplicate it.